libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -175,26 +175,26 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 {
 	yulAssert(m_scope, "");
 
-	int const numVariables = _varDecl.variables.size();
+	size_t const numVariables = _varDecl.variables.size();
 	int heightAtStart = m_assembly.stackHeight();
 	if (_varDecl.value)
 	{
 		std::visit(*this, *_varDecl.value);
-		expectDeposit(numVariables, heightAtStart);
+		expectDeposit(int(numVariables), heightAtStart);
```

Comment:
Maybe unrelated.

I think the type of `expectDeposit` needs to change from `void CodeTransform::expectDeposit(int _deposit, int _oldHeight) const` to `void CodeTransform::expectDeposit(size_t  _deposit, size_t  _oldHeight) const`

Reply:
Can you create a follow-up PR?

Reply:
Yes.